### ISSUE TITLE
LoadError: no such file to load -- liquid

### ISSUE THREAD

**Issue by kamatama41** [embulk/embulk#336]
I updated Embulk to v0.7.8 a while ago.
After that, It has occurred the following error.

Environment
- Embulk 0.7.7 -> 0.7.8
- Java 1.8.0_60
- MacOS X 10.10.5

```
% embulk run test.yml.liquid 
2015-11-09 18:12:01.096 +0900: Embulk v0.7.8
LoadError: no such file to load -- liquid
      require at org/jruby/RubyKernel.java:940
      require at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:54
  read_config at /Users/shinichi/.embulk/bin/embulk!/embulk/runner.rb:114
          run at /Users/shinichi/.embulk/bin/embulk!/embulk/runner.rb:55
          run at /Users/shinichi/.embulk/bin/embulk!/embulk/command/embulk_run.rb:296
        <top> at /Users/shinichi/.embulk/bin/embulk!/embulk/command/embulk_main.rb:2
      require at org/jruby/RubyKernel.java:940
       (root) at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
        <top> at file:/Users/shinichi/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51
```

**Comment by frsyuki** [comment]
Thank you for reporting. Fixed at 16cb2b6.

**Comment by kamatama41** [comment]
Thank you for the quick response!
